src/LowerPattern.jl

```diff
@@ -1,3 +1,9 @@
+# compute whether or not a constant pattern matches a value at runtime
+# For compat with Match.jl the order of arguments is reversed from @ismatch
```

Comment:
do we still want to hold to that? or just switch it and mark it as a breaking change in a release note somewhere?

Reply:
That's a great idea.  I doubt anyone overrides the internal API.  I'll update it in a separate PR to avoid merge hell.  Maybe change the function name as well.